src/pages/index.js

```diff
@@ -21,25 +21,55 @@ import {
   addCardModal,
   addCardFormElement,
   initialCards,
+  editAvatarButton,
+  avatarEditForm,
 } from "../utils/constants.js";
+import Api from "../components/Api.js";
+import PopupConfirmation from "../components/PopupConfirmation.js";
 
 const editFormValidator = new FormValidator(config, profileEditForm);
 const addFormValidator = new FormValidator(config, addCardFormElement);
+const avatarFormValidator = new FormValidator(config, avatarEditForm);
 
 editFormValidator.enableValidation();
 addFormValidator.enableValidation();
+avatarFormValidator.enableValidation();
 
-function createCard(item) {
-  const card = new Card(item, "#card-template", handleImageClick);
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "05177711-40d5-4e6a-abe7-ba1645f6efc9",
+    "Content-Type": "application/json",
+  },
+});
+
+function createCard(data) {
+  const card = new Card(
+    data,
+    "#card-template",
+    handleImageClick,
+    handleDeleteCard,
+    handleLikeClick
+  );
   return card.getView();
 }
 
+function renderCard(data) {
+  const card = createCard(data);
+  cardListEl.addItems(card);
+}
+
+api.getInitialCards().then((cards) => {
+  cards.forEach((data) => {
+    const card = createCard(data);
+    cardListEl.addItems(card);
```

Comment:
Please, use `renderItems` for rendering the array


You need to change it like this

```js

  renderItems(items) {
    items.forEach((item) => {
      this._renderer(item);
    });
  }
```

And use it here 

```js
api
  .getInitialCards()
  .then((cards) => {
    cardListEl.renderItems(cards);
  })
  .catch(console.error);

```

Reply:
Hello changing the index.js to the provided code above as broken the website. I do have that same code in my section.js file and use the code line on 78 of index.js to render the initial items. 
![image](https://github.com/user-attachments/assets/0cd85bd9-b505-475a-937f-f293a90e816c)